src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,276 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert BioGRID to BEL.
+
+Run this script with ``python -m bio2bel.sources.biogrid``
+
+The interaction information contained in `BioGRID <https://thebiogrid.org>`_ can be categorized into protein interactions, genetic interactions,
+chemical associations, and post-translational modifications. BioGRID includes information from major model organisms
+and humans.
+The file downloaded from BioGRID is a zip archive containing a single file formatted in PSI MITAB level 2.5 compatible
```

Comment:
link for PSI-MITAB level 2.5 spec

Reply:
also, it would be good to list right here some other data sources that are available in PSI-MITAB level 2.5 as well